### ISSUE TITLE
Add `pmros/cyclow` project

### ISSUE THREAD

**Issue by ghost** [michaelrambeau/bestofjs#74]
https://github.com/pmros/cyclow
A new reactive frontend framework for JavaScript

**Comment by michaelrambeau** [comment]
Hello Patricio @pmros 
Thank you very much for the suggestion.
`cyclow` project will be available on http://bestof.js.org/ very soon.

Just a suggestion: since there is a lot of competition in the `front-end frameworks` category, maybe you could mention 2 things in the project homepage:

* how light the package is once bundled (how many KB, is it a module with a small footprint compared to Cycle.js?)
* Does it perform well in the browser? (I guess it's fast since it uses `Snabbdom` to render the DOM)

**Comment by ghost** [comment]
Hello Michael,
Your feedback is highly appreciated! I'm gonna improve Cyclow website. I'll
include more information about Cyclow features.
Thank you!

Patricio

2017-04-08 1:06 GMT+02:00 Michael Rambeau <notifications@github.com>:

**Comment by ghost** [comment]
I can see Cyclow at http://bestof.js.org/projects/cyclow! Thank you!

If you need a Cyclow logo you can use
http://cyclow.js.org/img/favicon128.png
instead of my avatar (http://bestof.js.org/search/cyclow)

By the way, I've updated Cyclow website.
Thank you again!

**Comment by michaelrambeau** [comment]
Hello Patricio @pmros, 
Yes, it's online since this morning but I had to fix the npms score that not has been updated yet.

About the web site, good job! This is what I wanted to read about the project 👍 .

![image](https://cloud.githubusercontent.com/assets/5546996/24853919/8dcaf6ec-1e17-11e7-9178-1e7649621c80.png)

About the project icon, I will find a way to specify a specific URL but for now, only the project's owner avatar is displayed. Stay tuned!

Thank you for having starred the project, any feedback is welcome!
